app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/labtest/LabTestDetailViewModel.kt

```diff
@@ -122,20 +122,33 @@ class LabTestDetailViewModel @Inject constructor(
             labTestDetails.add(
                 0,
                 LabTestDetail(
+                    bannerTitle = R.string.results_are_pending,
                     bannerText = R.string.lab_test_detail_banner_1,
                     viewType = ITEM_VIEW_TYPE_LAB_TEST_BANNER
                 )
             )
         }
-
-        if (testStatus == R.string.pending) {
-            labTestDetails.add(
-                0,
-                LabTestDetail(
-                    bannerText = R.string.lab_test_detail_banner_2,
-                    viewType = ITEM_VIEW_TYPE_LAB_TEST_BANNER
+        when (testStatus) {
+            R.string.pending -> {
+                labTestDetails.add(
+                    0,
+                    LabTestDetail(
+                        bannerTitle = R.string.results_are_pending,
+                        bannerText = R.string.lab_test_detail_banner_2,
+                        viewType = ITEM_VIEW_TYPE_LAB_TEST_BANNER
+                    )
                 )
-            )
+            }
+            R.string.cancelled -> {
```

Comment:
Learn more is placed twice in the cancelled test result. Also please verify the text with the updated screen design.